k8s/charts/openebs/templates/deployment-maya-apiserver.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     chart: {{ template "openebs.chart" . }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
-    component: apiserver
+    openebs.io/component: maya-apiserver
```

Comment:
Should this be `openebs-apiserver` or `maya-apiserver` ?

Reply:
maya-apiserver